pkg/accounting/tally/tally_test.go

```diff
@@ -50,16 +55,22 @@ func TestOnlineNodes(t *testing.T) {
 		}
 	}
 	overlayServer := mocks.NewOverlay(nodes)
-	kad := &kademlia.Kademlia{}
+	rootdir, cleanup := mktempdir(t, "kademlia")
+	defer cleanup()
+	kad, err := kademlia.NewKademlia(node.IDFromString("foo"), []pb.Node{}, "127.0.0.1:8080", nil, &provider.FullIdentity{}, rootdir, 1)
```

Comment:
I can't remember if this endpoint supports :0 as the port - which would use a random open port (if it works)

Reply:
Or use `testplanet` which does the system setup.